src/CTA.WebForms2Blazor/ControlConverters/LabelControlConverter.cs

```diff
@@ -9,19 +11,31 @@ public class LabelControlConverter : ControlConverter
         protected override Dictionary<string, string> AttributeMap { 
             get
             {
-                throw new NotImplementedException();
+                return new Dictionary<string, string>()
+                {
+                    ["id"] = "id"
+                };
             } 
         }
-        protected override string BlazorName { get { throw new NotImplementedException(); } }
-        
-        public LabelControlConverter() : base()
-        {
-            
-        }
+        protected override string BlazorName { get { return "label"; } }
+
+        //protected override string NodeTemplate { get { return @"{2}"; } }
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            var labelText = "";
+            try
+            {
+                var textAttr = node.Attributes.AttributesWithName("text").Single();
+                labelText = textAttr.Value;
+            }
+            catch (InvalidOperationException ex)
+            {
+                //Todo: Throw warning about label with no text
+            }
```

Comment:
Same as above, we can replace try/catch if we set a default value when the text field is null/missing.